camera_models.py

```diff
@@ -644,4 +645,152 @@ def point_3d_2_pixel(self, point_3d, normalized=False):
         
         return self.out_to_numpy( out ), \
                self.out_to_numpy( theta <= self.fov_rad / 2.0 )
-    
\ No newline at end of file
+    
+
+
+

```

Comment:
Register this camera model to `CAMERA_MODELS` as https://github.com/castacks/mvs_utils/blob/84e619c59a3c6a80407b18460049aa8195ad5cb3/camera_models.py#L152